contents/articles/2017-12_object_fixation/index.md

```diff
@@ -0,0 +1,26 @@
+--- 
+title: Real-time object recognition using fixations to control prosthetic hand
+date: Fri Dec 15 2017 11:31:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Real-time object recognition with fixations to control grasps of a robotic prosthetic hand..."
+featured_img: "../../../../media/images/blog/object_detection.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/object_detection.jpg"
+---
+
+We are excited to share a project from a Pupil community member, [Jesse Weisberg]((https://github.com/jesseweisberg). Jesse has developed a plugin for Pupil that enables a prosthetic hand to form grasps based on what the wearer is fixating on in the scene in real-time on the CPU (10-15 fps)!
+
```

Comment:
Feel free to attach this to the end of previous paragraph or wherever you see fit.

This plugin was developed as part of a project called [Therabotics](https://www.jesseweisberg.com/therabotics/).  Therabotics was created by a team of graduate students at Washington University in St. Louis to explore how computer vision can be used to improve prosthetic hand control.

Reply:
@jesseweisberg Thanks - this has been added!